deeds-dapp-webapp/src/main/webapp/vue-app/dapp/layout/components/Navbar.vue

```diff
@@ -37,7 +37,7 @@
             class="px-0 full-height"
             link
             @click="openPage">
-            <h3 class="mx-2">{{ $t('page.marketplace') }}</h3>
+            <h3 class="mx-2 number-format">{{ $t('page.marketplace') }}</h3>
```

Comment:
`number-format` a class dedicated to number formatting which is used for text styling. could you fix this please.

Reply:
fixed!